### PR TITLE
Bundling

### PR DESCRIPTION
**Pull request by ianwremmel**
I *think* 75fa5e8 is the only commit that's not superseded by other PRs.

### CODE REVIEW

File: tooling/browserify/dot-browser.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/* eslint-disable no-invalid-this */
+
+const debug = require(`debug`)(`tooling:dot-browser`);
+const fs = require(`fs`);
+const path = require(`path`);
+const {wrap} = require(`lodash`);
```
Comment (bzang) [inline tooling/browserify/dot-browser.js]: would it be better to adapt the `const wrap = require(`lodash/wrap`);` syntax here as well just to be consistent with the code that supports tree shaking?

File: tooling/lib/build.js [hunk 2]
```diff
@@ -6,32 +6,104 @@ const {
...
+        dest: {
+          browser: path.resolve(`packages`, `node_modules`, packageName, filename.replace(`src`, `dist/browser`)),
+          main: path.resolve(`packages`, `node_modules`, packageName, filename.replace(`src`, `dist/main`)),
+          module: path.resolve(`packages`, `node_modules`, packageName, filename.replace(`src`, `dist/module`))
+        }
+      });
+    }
+  }
+
+  const config = JSON.parse(await readFile(path.join(process.cwd(), `.babelrc`)));
```
Comment (bzang) [inline tooling/lib/build.js]: Would using the sync version of `readFile` (and other fs methods) make more sense than doing async/await?

File: tooling/browserify/dot-browser.js (hunk 1 above)
Comment (ianwremmel) [inline tooling/browserify/dot-browser.js]: yea, that's probably a good idea.

File: tooling/lib/build.js (hunk 2 above)
Comment (ianwremmel) [inline tooling/lib/build.js]: maybe. `await` is used all over tooling, so it's not trivial to take it out; I think at this point remaining consistent with what's there seems like the way to go.

**Pull request closed without merging**